test/unit/lib/i18n/i18n.spec.coffee

```diff
@@ -0,0 +1,184 @@
+fdescribe 'i18n', ->
+
+  core = require( 'src/index' )
+  instance = fooInstance = testInstance = null
+
+  # Cached vars
+  LANG_ES = 'es'
+  LANG_EN = 'en'
+
+  TEXT_EN_TITLE = 'Title'
+  TEXT_EN_TXT = 'Text'
+  TEXT_EN_NOT_TRANSLATED = 'Not translated'
+  TEXT_EN_WITH_PARAMS = 'Mr. %1 got %2 gems. Did I said %1?'
+  TEXT_ES_TITLE = 'Título'
+  TEXT_ES_TXT = 'Texto'
+  TEXT_ES_WITH_PARAMS = 'El Sr. %1 tiene %2 gemas. ¿Dije %1?'
+
+  # Fake data
+  textsEN = {
+    'TEST_TITLE': TEXT_EN_TITLE
+    'TEST_TXT': TEXT_EN_TXT
+    'TEST_NOT_TRANSLATED': TEXT_EN_NOT_TRANSLATED
+    'TEST_WITH_PARAMS': TEXT_EN_WITH_PARAMS
+  }
+  textsES = {
+    'TEST_TITLE': TEXT_ES_TITLE
+    'TEST_TXT': TEXT_ES_TXT
+    'TEST_WITH_PARAMS': TEXT_ES_WITH_PARAMS
+  }
+
+  isFunction = ( it ) ->
+    return Object.prototype.toString.call(it) is '[object Function]'
```

Comment:
As far as this is coffeescript, the parenthesis spaces must match the conventions